jsf-showcase-webapp/src/test/java/com/liferay/faces/test/showcase/TesterBase.java

```diff
@@ -54,7 +54,7 @@
 
 	static {
 
-		String projectVersion = System.getProperty("integration.showcase.version");
+		String projectVersion = System.getProperty("integration.showcase.version", "3.0.0-SNAPSHOT");
```

Comment:
@ngriffin7a, when you backport this, please change `"3.0.0-SNAPSHOT"` to `"2.0.0-SNAPSHOT"`.